mne_bids/mne_bids.py

```diff
@@ -654,10 +654,13 @@ def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
                          % (''.join(ALLOWED_EXTENSIONS), ext))
 
     # Copy the imaging data files
-    # Re-save FIF files to fix the file pointer for files with multiple parts
-    # This is WIP, see: https://github.com/mne-tools/mne-python/pull/5470
     if ext in ['.fif']:
-        raw.save(raw_file_bids, overwrite=overwrite)
+        n_rawfiles = len(read_raw_fif(raw_fname).filenames)
+        if n_rawfiles > 1:
+            split_naming = 'bids'
+        else:
+            split_naming = 'neuromag'
+        raw.save(raw_file_bids, split_naming=split_naming, overwrite=overwrite)
```

Comment:
@teonbrooks can we make `L658-662` a private function which uses `check_version`. If the `mne` version is less than `0.16dev0` _and_ `n_rawfiles > 1`, then raise an error asking the user to upgrade. Otherwise, continue as usual. That way, we can merge this PR without having to wait for the `mne` release

Reply:
+1 to this suggestion

Reply:
done